ngrinder-controller/src/main/java/org/ngrinder/agent/controller/MonitorDownloadController.java

```diff
@@ -35,23 +33,25 @@
 /**
  * Monitor Download Controller.
  *
- * @author Tobi
  * @since 3.0
  */
 @Controller
 @RequestMapping("/monitor/download")
-public class MonitorDownloadController extends BaseController {
+public class MonitorDownloadController {
 
-	@Autowired
 	private AgentPackageService agentPackageService;
 
-	@Autowired
 	private Config config;
 
-	@Autowired
-	@Qualifier("monitorPackageHandler")
 	private MonitorPackageHandler monitorPackageHandler;
 
+	public MonitorDownloadController(AgentPackageService agentPackageService, Config config,
+									 @Qualifier("monitorPackageHandler") MonitorPackageHandler monitorPackageHandler) {
+		this.agentPackageService = agentPackageService;
+		this.config = config;
+		this.monitorPackageHandler = monitorPackageHandler;
+	}
```

Comment:
Postponed to apply lombok here because lombok constructor cannot have `@Qualifier` annotation. I'm gonna apply lombok and not use `@Qualifier` annotation.